lib/commons/dom/is-focusable.js

```diff
@@ -70,3 +70,19 @@ dom.isNativelyFocusable = function(el) {
 	}
 	return false;
 };
+
+/**
+ * Determines if an element is in the focus order, but would not be if its
+ * tabindex were unspecified.
+ * tabindex
+ * @method insertedIntoFocusOrder
+ * @memberof axe.commons.dom
+ * @instance
+ * @param {HTMLElement} el The HTMLElement
+ * @return {Boolean} True if the element is in the focus order but wouldn't be
+ * if its tabindex were removed. Else, false.
+ */
+dom.insertedIntoFocusOrder = function(el) {
```

Comment:
Either this should be in its own file, or the test file you created for it should be merged with the existing tests for is-focusable.js. It's best to have the test files match the name of the files they're testing.

Reply:
Sure, the same logic should apply to [dom.isNativelyFocusable](https://github.com/dequelabs/axe-core/blob/develop/lib/commons/dom/is-focusable.js#L48), right?

Reply:
Yep